.azure-pipelines/official-build.yml

```diff
@@ -38,7 +38,7 @@ stages:
   variables:
   - name: CACHE_MODE
     value: wcache
-  - template: azure-pipelines-repd-build-variables.yml
+  - template: azure-pipelines-repd-build-variables.yml@buildimage
```

Comment:
Need to change to .azure-pipelines/azure-pipelines-repd-build-variables.yml@buildimage